src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/integration/AppointmentIntegrationTest.kt

```diff
@@ -0,0 +1,39 @@
+package uk.gov.justice.digital.hmpps.whereabouts.integration
+
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.jdbc.core.JdbcTemplate
+import org.springframework.transaction.annotation.Transactional
+import java.time.LocalDate
+
+@Transactional
+class AppointmentIntegrationTest : IntegrationTest() {
+
+  @Autowired
+  lateinit var jdbcTemplate: JdbcTemplate
```

Comment:
Not sure it's used but if it's going to be might be better off in the base class